docs/ext/commands/slash_commands.rst

```diff
@@ -184,9 +184,9 @@ The type of the option is determined by the range bounds, with the option being
 
 .. note::
 
-    Type checker support for :class:`~ext.commands.Range` is limited. Pylance/Pyright seem to handle it correctly;
```

Comment:
maybe move this warning to a separate section and reference it from here and from the `String Lengths` section below? Not sure

Reply:
I did something similar to this with https://github.com/DisnakeDev/disnake/pull/593/commits/d60906a486cd58fd82586e4258b1400844203461: viewable here https://disnake--593.org.readthedocs.build/en/593/ext/commands/slash_commands.html#type-checker-mypy-plugin